src/test/java/org/wildfly/test/integration/vdx/standalone/MessagingTestCase.java

```diff
@@ -30,45 +47,123 @@ public void modifyWsdlAddressElementWithNoValue() throws Exception {
+        assertContains(errorLog, "convert \\\"not-int-value\\\" to [INT]");
+    }
+
+    /*
+    * provide invalid type to server-id to in-vm-acceptor - put long to int
+    */
+    @Test
+    @ServerConfig(configuration = "standalone-full-ha.xml", xmlTransformationGroovy = "messaging/InvalidTypeSetLongInServerIdInAcceptor.groovy",
+            subtreeName = "messaging", subsystemName = "messaging-activemq")
+    public void testLongInIntServerIdInAcceptor() throws Exception {
+        container().tryStartAndWaitForFail();
+        String errorLog = container().getErrorMessageFromServerStart();
+        assertContains(errorLog, "OPVDX001: Validation error in standalone-full-ha.xml ---------------------------");
+        assertContains(errorLog, "<in-vm-acceptor name=\"in-vm\" server-id=\"214748364700\"/>");
+        assertContains(errorLog, "^^^^ Wrong type for 'server-id'. Expected [INT] but was STRING. Couldn't");
+        assertContains(errorLog, "convert \\\"214748364700\\\" to [INT]");
+    }
 
+    /*
+    * provide invalid type - too long value to long type
+    */
+    @Test
+    @ServerConfig(configuration = "standalone-full-ha.xml", xmlTransformationGroovy = "messaging/TooLongValueForMaxSizeBytes.groovy",
+            subtreeName = "messaging", subsystemName = "messaging-activemq")
+    public void testTooLongValueForLongTypeInMaxSizeBytes() throws Exception {
+        container().tryStartAndWaitForFail();
+        String errorLog = container().getErrorMessageFromServerStart();
+        assertContains(errorLog, "OPVDX001: Validation error in standalone-full-ha.xml");
+        assertContains(errorLog, "<address-setting name=\"#\" dead-letter-address=\"jms.queue.DLQ\" " +
+                "expiry-address=\"jms.queue.ExpiryQueue\" max-size-bytes=\"1048576000000000000000000000000000000000\" " +
+                "page-size-bytes=\"2097152\" message-counter-history-day-limit=\"10\" redistribution-delay=\"1000\" " +
+                "address-full-policy=\"PAGE\"/>");
+        assertContains(errorLog, " ^^^^ Wrong type for 'max-size-bytes'. Expected [LONG] but was STRING.");
+        assertContains(errorLog, "Couldn't convert \\\"1048576000000000000000000000000000000000\\\" to");
+        assertContains(errorLog, "[LONG]");
     }
 
+    /*
+    * provide invalid type - test double in long
+    */
+    @Test
+    @ServerConfig(configuration = "standalone-full-ha.xml", xmlTransformationGroovy = "messaging/DoubleInLongTypeInMaxSizeBytes.groovy",
+            subtreeName = "messaging", subsystemName = "messaging-activemq")
+    public void testDoubleInLongTypeInMaxSizeBytes() throws Exception {
+        container().tryStartAndWaitForFail();
+        String errorLog = container().getErrorMessageFromServerStart();
+        assertContains(errorLog, "OPVDX001: Validation error in standalone-full-ha.xml ---------------------------");
+        assertContains(errorLog, "<address-setting name=\"#\" dead-letter-address=\"jms.queue.DLQ\" " +
```

Comment:
Is it safe to have such long assert ? Can't it change in time ?

It can be OK to have it, just need to know your reasoning.